test/e2e/quadlet/group-add.container

```diff
@@ -0,0 +1,7 @@
+## assert-podman-args "--group-add=keep-groups"
+## assert-podman-args "--group-add=users"
```

Comment:
The code does not use `=` so the check should get two strings instead
```suggestion
## assert-podman-args "--group-add" "keep-groups"
## assert-podman-args "--group-add" "users"
```

Reply:
Actually, the manpage lists both formats. I guess both work?

Reply:
Yes. The argument parser can handle both (equal or space)